doc/source/visualization.rst

```diff
@@ -2,25 +2,31 @@
 .. _visualization:
 
 .. ipython:: python
+   :flake8-add-ignore: E702, E703
    :suppress:
 
    import numpy as np
    import pandas as pd
+
+   import matplotlib  # noqa: F401
+   # matplotlib.style.use('default')
+   import matplotlib.pyplot as plt
+
    np.random.seed(123456)
    np.set_printoptions(precision=4, suppress=True)
    pd.options.display.max_rows = 15
-   import matplotlib
-   # matplotlib.style.use('default')
-   import matplotlib.pyplot as plt
    plt.close('all')
 
+
 *************
 Visualization
 *************
 
 We use the standard convention for referencing the matplotlib API:
 
 .. ipython:: python
+   :flake8-group: None
+   :flake8-set-ignore: F401
```

Comment:
if there are unused imports please remove them, and if really needed, better a `noqa` in the import